string CADFileName
string CADMarkerName
string ScanFileName
string ScanMarkerName
---
bool ReturnStatus
string ReturnMessage
